core/src/replay/mod.rs

```diff
@@ -184,6 +186,15 @@ impl Stream for Historator {
                                  execution started events",
                     )
                     .to_string();
+                if self.seen_run_ids.contains(&run_id) {
```

Comment:
What is the situation where we run the same run ID over and over?

I wonder if we should break here. People may have a history file they are altering a few ways to run on the same replayer. I'd rather break on them than let this invalid set through.

Reply:
If they literally pass in multiple histories with the same run id